content/en/Getting started/track-credits.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Track Your Credits"
+linkTitle: "Track Your Credits"
+weight: 380
+description: >
+  View and track your credits in the dashboard.
+---
+
+{{% pageinfo %}}
+Navigate to the **Credits** tab to analyze how your company uses Cobalt [credits](https://www.cobalt.io/pentest-pricing#cobaltcredits).
+{{% /pageinfo %}}
+
+Here, you can see the following:
+
+- Current credit balance
+- Contract end date showing when your credits expire
+- History of pending and completed credit transactions
+  - You may also see your {{% ptaas-tier %}} shown as "Subscription Plan".
```

Comment:
@molfinn Changed to `You may also see your {{% ptaas-tier %}} shown as "Subscription Plan".`